src/main/java/duke/storage/Storage.java

```diff
@@ -0,0 +1,82 @@
+package duke.storage;
+
+import duke.task.*;
+import duke.exception.DukeException;
+import java.io.FileNotFoundException;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.Scanner;
+import java.io.File;
+import java.io.FileWriter;
+
+public class Storage {
+    private String filePath;
+
+    private ArrayList<Task> list;
+
+    public Storage(String fp) {
+        this.filePath = fp;
+    }
+
+    public ArrayList<Task> load() throws DukeException {
+        list = new ArrayList<>();
+        try {
+            File f = new File(filePath);
+            Scanner sc = new Scanner(f);
+            while (sc.hasNextLine()) {
+                String item = sc.nextLine();
+                String type = item.substring(0, 3);
+                String status = item.substring(3, 6);
+                String desc;
+                String date = "";
+                if (item.contains("(")) {
+                    int dateDescInd = item.indexOf("(");
+                    desc = item.substring(7, dateDescInd - 1);
+                    date = item.substring(dateDescInd + 5, item.length() - 1);
+                } else {
+                    desc = item.substring(7);
+                }
+            Task t = null;
+            switch (type) {
+                case "[D]" : t = new Deadline(desc, date); break;
+                case "[T]" : t = new ToDo(desc); break;
+                case "[E]" : t = new Event(desc, date); break;
+                default:
+                    System.out.println("Unknown event type encountered.");
+            }
+            if (status.equals("\u2713")) {
```

Comment:
Perhaps could consider non-unicode characters.

At least on my computer, the done attribute is not preserved when the program reloads (i.e. after marking a task as done and exiting the program, all tasks marked as done are reset as undone when the program starts up again)

Does this phenomenon occur on the device you're running the program on too?